HISTORY.rst

```diff
@@ -3,6 +3,25 @@ Release notes
 =============
 
 
+2.9.1 (2021-03-22)
+------------------
+
+* This is relatively large change, it could have justified a major version bump,
```

Comment:
I feel like we should adhere to semantic versioning as strictly as possible. If this changes behaviour, we should bump the major version. There's no harm in incrementing numbers quickly; but it could be annoying if anyone is dependent on certain version schemes.

I would bump to 2.10.

Reply:
Sold, let's bump to 3.0... it's a big enough change, and major version numbers are not rare or anything :)
So no reason not to, plus it'll nicely mark the PEP-440 compliance.

Changed release notes to 3.0, and added more info (removed the old paragraph explaining why no major bump)